src/test/java/io/confluent/idesidecar/restapi/resources/ConnectionsResourceTest.java

```diff
@@ -464,7 +464,7 @@ void updateConnectionWithMultipleValidationErrors() {
 
     var badSpec = new ConnectionSpec(
         "c3", "Connection name changed!", ConnectionType.PLATFORM,
-        new CCloudConfig("org-id"));
+        new CCloudConfig("org-id"), null);
```

Comment:
Nit: proper indentation and line feeds helps _a lot_ here:
```suggestion
        new CCloudConfig("org-id"),
        null
    );
```

Reply:
done.